p5/core/transforms.py

```diff
@@ -154,12 +154,21 @@ def scale(sx, sy=None, sz=None):
 
 
 def apply_matrix(transform_matrix):
-    """Apply the given matrix to the sketch's transform matrix..
+    """
+    Apply the given matrix to the sketch's transform matrix.
 
     :param transform_matrix: The new transform matrix.
-    :type transform_matrix: np.ndarray (or a 4×4 list)
+    :type transform_matrix: (3 * 3) if skia is the renderer else (4 * 4) np.ndarray or list
```

Comment:
I will leave a TODO for now, since skia supports [3 * 3 matrix transformations only](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/graphics/skiasharp/transforms/matrix#the-reason-for-the-3-by-3-matrix)

Reply:
Oops I forgot skia is a 2D only renderer. Then yes, supporting only 3*3 matrices make sense. We could probably also improve the documentation to say that 3x3 is expected for 2D renderers and 4x4 is expected for 3D renderers.